dist-base/README.adoc

```diff
@@ -0,0 +1,49 @@
+= Apicurio Registry Operator Distribution Bundle
+
+This directory contains installation files for the Apicurio Registry Operator, with example `ApicurioRegistry`
+custom resource files. In addition, full operator documentation and license information is also included.
```

Comment:
Suggest uppercase O for Operator in all cases to match Kubernetes standards. Occurs in a few places in this readme.